tsl/src/bgw_policy/reorder_api.c

```diff
@@ -105,6 +106,19 @@ check_valid_index(Hypertable *ht, Name index_name)
 	ReleaseSysCache(idxtuple);
 }
 
+Datum
+policy_reorder_check(PG_FUNCTION_ARGS)
+{
+	if (PG_NARGS() != 1 || PG_ARGISNULL(0))
+		PG_RETURN_VOID();
```

Comment:
Same here. Might be good to add a test that this does not cause a crash or something else.